### DISCUSSION TITLE
New Styles ?

### DISCUSSION THREAD

**Discussion by xcreidcx** [lllyasviel/Fooocus#3514] [General]
Hi ! I was just curious if there will be new styles coming soon? If not thats okay just was curious :)

**Comment by Yogesh-DevHub** [comment]
Hi there,
I think, this will may make you even more curious: https://github.com/lllyasviel/Fooocus/discussions/3504

**Reply by xcreidcx** [reply]
Sorry I didn't fully understand it after reading it a couple of times since English isn't my first language :(

**Reply by Yogesh-DevHub** [reply]
The guy who manages fooocus appears to be busy with other things, I don't think styles will be introduced.
Styles are essentially additional prompts added around your original prompt, which you can add on your own. There is a JSON file, you can edit in Notepad and add your own style, just try to copy paste an existing one and make changes to it. 
The video explains about styles: https://youtu.be/TVBYtIEcpU4?t=249&feature=shared

Note: The data (styles) in this JSON file are listed in curly brackets like this { }, each bracket has a comma at the end, but the last bracket shouldn't have one, like this [ { }, { }, { }, { }, { } ]
